includes/reader-activation/class-reader-activation.php

```diff
@@ -1781,5 +1781,72 @@ public static function dont_force_registration_for_existing_woo_users( $data ) {
 
 		return $data;
 	}
+
+	/**
+	 * Verify the given user, bypassing the need to complete the email-based verification flow.
+	 */
+	public static function cli_setup_ras() {
+		if ( self::is_ras_campaign_configured() ) {
+			\WP_CLI::error( __( 'RAS is alreaady configured for this site.', 'newspack-plugin' ) );
+		}
+
+		if ( ! class_exists( '\Newspack_Popups_Presets' ) ) {
+			\WP_CLI::error( __( 'Newspack Campaigns plugin not found.', 'newspack-plugin' ) );
+		}
+
+		
+		if ( ! class_exists( '\Newspack_Newsletters_Subscription' ) ) {
+			\WP_CLI::error( __( 'Newspack Newsletters plugin not found.', 'newspack-plugin' ) );
+		}
+
+		if ( \is_wp_error( \Newspack_Newsletters_Subscription::get_lists() ) ) {
+			\WP_CLI::error( __( 'Newspack Newsletters provider not set.', 'newspack-plugin' ) );
+		}
+
+		$result = \Newspack_Popups_Presets::activate_ras_presets();
+		
+		if ( ! $result ) {
+			\WP_CLI::error( __( 'Something went wrong. Please check for required plugins and try again.', 'newspack-plugin' ) );
+			exit;
+		}
+
+		\WP_CLI::success( __( 'RAS enabled with default prompts.', 'newspack-plugin' ) );
+	}
+
+	/**
+	 * Verify the given user, bypassing the need to complete the email-based verification flow.
+	 *
+	 * @param array $args Positional args.
+	 * @param array $assoc_args Associative args.
+	 */
+	public static function cli_verify_reader( $args, $assoc_args ) {
+		$user_id_or_email = ! empty( $args ) ? reset( $args ) : false;
+		if ( ! $user_id_or_email ) {
+			\WP_CLI::error( __( 'Please provide a user ID or email address.', 'newspack-plugin' ) );
+			exit;
+		}
+
+		$get_by = is_numeric( $user_id_or_email ) ? 'id' : 'email';
+		$user   = \get_user_by( $get_by, $user_id_or_email );
+
+		if ( ! $user ) {
+			\WP_CLI::error( __( 'User not found.', 'newspack-plugin' ) );
+			exit;
+		}
+
+		self::set_reader_verified( $user );
+
+		if ( method_exists( '\Newspack_Newsletters_Subscription', 'set_email_verified' ) ) {
+			\Newspack_Newsletters_Subscription::set_email_verified( $user->ID );
+		}
```

Comment:
This bit is superfluous since [newsletters verifies](https://github.com/Automattic/newspack-newsletters/blob/11efaa7841374e0b8abe52a44bbe77f30f645e23/includes/class-newspack-newsletters-subscription.php#L39) on a hook triggered on `set_reader_verified()`.

Reply:
Ah, nice! I missed that

Reply:
Removed in 20d95ab5